src/python/WMCore/WMBS/MySQL/Create.py

```diff
@@ -65,6 +65,26 @@ def __init__(self, logger = None, dbi = None, params = None):
           """CREATE UNIQUE INDEX uniq_wmbs_file_run_lumi on
              wmbs_file_runlumi_map (fileid, run, lumi)"""
 
+        #
+        # Insert LogCollect site to account for LogCollect output
+        #
+        self.inserts["99_location_logcollect_1"] = \
```

Comment:
Well, then you should actually report the tarBall stagout to T2_CH_CERN (and not report it when that optional stageout fails), which you don't do. As-is the LogCollect stageout and what it is reporting doesn't make sense. I am going to talk to @ticoann today to figure out what we are actually supposed to be reporting here.

And btw, we haven't done this for years, this reporting of LogCollect output is new, we have never done this before.